datastore-usersetup.py

```diff
@@ -7,7 +7,7 @@
 import psycopg2
 import re
 import sys
-from urlparse import urlparse
+from urllib.parse import urlparse
```

Comment:
You didn't run `futurize` on this, right?  You just updated the library call?

Reply:
Correct, but we confirmed that the script ran to completion.